scot/ooapi.py

```diff
@@ -100,6 +106,12 @@ def __init__(self, var, locations=None, reducedim=None, nfft=512, fs=2, backend=
         except TypeError:
             self.var_ = var
 
+    def __repr__(self):
+        s = ("Workspace(var={}, locations={}, reducedim={}, nfft={}, fs={}, "
+             "backend={})".format(self.var_, self.locations_, self.reducedim_,
+                                  self.nfft_, self.fs_, self.backend_))
```

Comment:
This is a problem because the name is separated from the backend. We could make a cusom class (derived from `dict`) with a `name` property.